extras/AFC_functions.py

```diff
@@ -592,6 +597,149 @@ def _register_klipper(self, show_macros, macro_name, command, description, optio
                     self.config.access_tracking[(name.lower(), option.lower())] = 1
             self.printer.load_object(self.config, name)
 
+    cmd_AFC_TEST_LANES_help = 'Run load/unload tests on specified lanes'
+    def cmd_AFC_TEST_LANES(self, gcmd):
+        """
+        Run load/unload tests on specified lanes. This command allows users to test the loading and unloading
+        mechanisms to serve as a stress test / diagnostic tool for the AFC system with an interactive prompt.
+
+        Usage
+        -----
+        `AFC_TEST_LANES`
+
+        Example
+        -----
+        ```
+        AFC_TEST_LANES
+        ```
+        """
+
+        no_lanes_loaded = not any(lane.load_state for lane in self.afc.lanes.values())
+
+        if no_lanes_loaded:
+            self.logger.info('No lanes loaded, please load lanes before running tests.')
+            return
+
+        if self.afc.current is not None:
+            self.logger.info('Toolhead must be unloaded to run tests.')
+            return
+
+        prompt = AFCprompt(gcmd, self.logger)
+        buttons = []
+        title = 'AFC Test Lanes'
+        text = ('The following prompts will allow you to test the loading and unloading of your lanes.'
+                ' Lanes must be loaded before running tests.')
+
+        for index, (key, item) in enumerate(self.afc.lanes.items()):
+            if item.load_state:
```

Comment:
Only show lane if both sensors are triggered
```suggestion
            if item.load_state and item.prep_state:
```